test_main.py

```diff
@@ -1,6 +1,8 @@
-from .repositories import create_book, get_books, get_book, update_book, delete_book
+from sqlalchemy import inspect
+
 from .models import BookIn
-from sqlalchemy import create_engine, inspect
+from .repositories import create_book, delete_book, get_book, get_books, update_book
+
 # from . import TEST_BOOKS
 
 # Test data constants
```

Comment:
```suggestion
```suggestion
    {"title": "Carrie", "author": "Stephen King", "desc": "A horror novel."},
    {"title": "Ready Player One", "author": "Ernest Cline", "desc": "A sci-fi novel."}
```
```